logger/log.go

```diff
@@ -190,6 +200,46 @@ func (l *fileLogger) logDirect(label, format string, v ...interface{}) int {
 	return len(entry)
 }
 
+func (l *fileLogger) archivePurge(fname string) {
```

Comment:
Is fname a dir?

Reply:
Fully qualified name of the active log file.  So both the base logfile name and the directory where all the backups are being stashed derived from that.